CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.15.1)
+
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
+    execute_process(
+        COMMAND xcrun --sdk macosx --show-sdk-path
+        OUTPUT_VARIABLE CMAKE_OSX_SYSROOT
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+    SET(CMAKE_Swift_FLAGS "${CMAKE_Swift_FLAGS} -sdk ${CMAKE_OSX_SYSROOT}")
+endif()
```

Comment:
Please dont modify `CMAKE_Swift_FLAGS`, its meant to be used by users only.  You can use `append_compile_options` with a generator expression to add the flags if you must.  (L18 has an example of the generator expression).

Reply:
Hm, the generator expression doesn't work when CMake is trying to check for the Swift compiler. Is there any other way to fix that?